tools/bin/acceptance_test_kube_helm.sh

```diff
@@ -65,7 +64,7 @@ if [ -n "$CI" ]; then
 #  trap "mkdir -p /tmp/kubernetes_logs && write_all_logs" EXIT
 fi
 
-kubectl port-forward svc/airbyte-server-svc 8001:8001 &
+kubectl port-forward svc/airbyte-airbyte-server-svc 8001:8001 &
```

Comment:
when did the svc name change to `{release-name}-airbyte-server-svc`?  Do you know the reasoning behind this change?

I guess one nit is that the svc names can end up looking a bit awkward with this new naming format (such as `airbyte-airbyte-server-svc` or `dev-airbyte-airbyte-server-svc` depending on what name someone chooses for the helm release).